custom_model_runner/setup.py

```diff
@@ -68,7 +68,7 @@
         "datarobot_drum.resource.pipelines": ["*"],
         "datarobot_drum.resource.default_typeschema": ["*.yaml"],
     },
-    scripts=["bin/drum", "../drapps/drapps.py"],
+    scripts=["bin/drum", "../drapps/drapps.py", "../drapps/drapps"],
```

Comment:
why do we have both?  can we keep the new one and get rid of the old one?

Reply:
No, we can't.
This script `drapps.py` is outside of the package, so it is not packaged automatically. I have to put it into scripts.